source/docs/articles/sites/cloudflare-cdn-and-dns.md

```diff
@@ -54,3 +54,25 @@ In Drupal, no modules are required to use CloudFlare, but there is an optional [
 
 https://support.cloudflare.com/hc/en-us/articles/200169456-Why-is-JavaScript-or-jQuery-not-working-on-my-site-
 
+## FAQ
```

Comment:
@nataliejeremy @primerg I suggest the following:
- Replace h2 **Known Limitations** in line 47 above with **Considerations**
- Remove the **FAQ** section added here, instead add an entry for **Rocket Loader** to match existing style and flow.

Thoughts? Thanks for the PR @primerg - much appreciated!